guides/common/modules/con_active-directory-with-cross-forest-trust.adoc

```diff
@@ -5,7 +5,7 @@ Kerberos can create `cross-forest trust` that defines a relationship between two
 A domain forest is a hierarchical structure of domains; both AD and {FreeIPA} constitute a forest.
 With a trust relationship enabled between AD and {FreeIPA}, users of AD can access Linux hosts and services using a single set of credentials.
 ifdef::satellite[]
-For more information on cross-forest trusts, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-a-cross-forest-trust-between-idm-and-ad_planning-identity-management[Planning a cross-forest trust between IdM and AD] in _{RHEL} Planning Identity Management_.
+For more information on cross-forest trusts, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/planning_identity_management/planning-a-cross-forest-trust-between-idm-and-ad_planning-identity-management[Planning a cross-forest trust between IdM and AD] in _{RHEL} 9 guide_ or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-a-cross-forest-trust-between-idm-and-ad_planning-identity-management[Planning a cross-forest trust between IdM and AD] in _{RHEL} 8 guide_.
```

Comment:
Given the repetition, should we have an attribute for the RHEL guide URL? So `https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux`.

Reply:
I am OK anyways.
@maximiliankolb @asteflova your views?

Reply:
I like the idea!